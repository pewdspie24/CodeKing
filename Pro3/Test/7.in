100
48 14233457
885993 566719
824085 140479
336080 733102
951331 987495
223766 399301
540874 169885
365114 966173
553806 387851
443059 638100
715396 750248
311522 34720
200452 219886
945239 937729
837081 242004
680475 744391
642335 806730
108554 601267
793275 122401
482511 538175
167943 240977
182810 215459
529499 85036
636991 746777
75684 812697
865478 272613
286381 423652
634875 651918
379190 803946
780188 463677
828861 817370
561694 507160
990651 627338
817001 908243
935128 453915
606432 250034
130984 252110
686125 382977
138400 968017
126893 486363
966651 270413
672981 460711
281909 371425
211765 154427
954796 277893
897412 607962
283529 84816
169845 385734
888933 608700
