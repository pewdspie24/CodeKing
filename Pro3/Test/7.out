21
19
18
16
15
13
12
11
9
8
6
5
3
2
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
